### PR TITLE
Algem ads on dApp Staking UI implementation

### PR DESCRIPTION
**Pull request by sirius651**
**Pull Request Summary**

> Describe what changes this pull request makes to the repository

**Check list**
- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

screencapture-localhost-8080-2022-07-20-15_37_05

스크린샷 2022-07-20 오후 3 38 37

### CODE REVIEW

**Comment by bobo-k2** [comment]
Header and footer are not aligned properly, at least on my machine (Chrome in mobile device mode) and phone

image
image

File: src/components/dapp-staking/AlgemPanel.vue [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<template>
```
Comment (impelcrypto) [inline src/components/dapp-staking/AlgemPanel.vue:1]: Let's define texts in i18n file.

**Review by impelcrypto — CHANGES_REQUESTED** [review]

File: src/components/dapp-staking/AlgemPanel.vue [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          <a
+            href="https://docs.algem.io/algem-protocol/liquid-staking/liquid-staking-faq"
+            target="_blank"
+            ><span class="icon-external"> <astar-icon-external-link /> </span
+          ></a>
+        </li>
+      </ul>
+    </div>
+    <div class="text--final">Enjoy The Magic of Liquid Staking And Seize The Extra Earnings!</div>
+    <a href="https://app.algem.io/liquid-staking" target="_blank">
```
Comment (impelcrypto) [inline src/components/dapp-staking/AlgemPanel.vue:57]: `cursor: pointer` should not appear outside of the button.
image

**Comment by Kahonnohak** [comment]
Thanks! looking good:)

Please kindly take a look.

- The wallet drawer closes instead when I wanted to accept the term.
https://gyazo.com/491bbc2eedbb99328c93c2fb1984e061


Design side

** Tab **
- Status
-- Default (not selected) -> change the font weight to 500
-- Hover -> Change the font weight to 700
-- Focus (selected) -> have the blue line


-Styling
-- Please make the blue line round

-- The architecture is this : 
Screenshot 2022-07-21 at 11 20 14
--- please add 16px side padding
--- fix the width to 240 for this page (mobile 160px)
 
-- When changing the statuses, it's dancing. 
https://gyazo.com/17082dd1f5b4ef74259b3ca3222a0922

-- Animation: could we have the same animation when we hover transfer modal? (I couldn't find the code for that modal)


**Algem Ads**

- Please add a radius same as contents above - > .5rem

- Risk warning area
-- Could you please add a yellow underline when hovering, also make the icon yellow as default, please.  
-- Let's make the link icon smaller 32->24 and could you adjust the line height please?
Screenshot 2022-07-21 at 11 36 33

- I can see the line here and the background image is repeating? prefer to use just image if there is no problem.
Screenshot 2022-07-21 at 11 38 17



I have to go now but I would like to add a few adjustments on mobile as well. I'll come back for this in the afternoon.

**Review by impelcrypto — CHANGES_REQUESTED** [review]

**Comment by impelcrypto** [comment]
Let's fix the styling for mobile devices.

image

image

**Comment by Kahonnohak** [comment]
For the mobile:

This is the design - I think the design is not ready for mobile yet?
Algem on dApp Staking V1 mobile

**Comment by sirius651** [comment]
스크린샷 2022-07-21 오후 5 11 36

Made a fix for mobile and feedbacks above. please take a look, and let me know if something missed. thanks 
@Kahonnohak @impelcrypto

**Review by impelcrypto — APPROVED** [review]

**Comment by Kahonnohak** [comment]
Looking great!

Sorry I missed one thing - the image. We need to keep the ratio not shrink. You can place the image in the centre (sorry I placed it on the left) and fix the height 100% I think? so the width would be trimmed? I hope you got what I mean, otherwise I'll explain on the screen. 

With the risk warning - Sorry I didn't explain enough. 

- The link is the bottom one, not the top one. 
- see the image, with Figma I cannot change the colour of the underline but please keep the yellow for hovering. 
- And I would like the icon at the end of the sentence. And could you please make the icon grey again... it was a bit odd sorry!

Screenshot 2022-07-21 at 21 24 22

**Comment by sirius651** [comment]
> Looking great!
> 
> Sorry I missed one thing - the image. We need to keep the ratio not shrink. You can place the image in the centre (sorry I placed it on the left) and fix the height 100% I think? so the width would be trimmed? I hope you got what I mean, otherwise I'll explain on the screen.
> 
> With the risk warning - Sorry I didn't explain enough.
> 
> * The link is the bottom one, not the top one.
> * see the image, with Figma I cannot change the colour of the underline but please keep the yellow for hovering.
> * And I would like the icon at the end of the sentence. And could you please make the icon grey again... it was a bit odd sorry!

I believe fixed what you mentioned, could you check again @Kahonnohak

**Comment by Kahonnohak** [comment]
Awesome thanks! this looks good to me! I'll just share this with Shun before approving. Thanks!!

**Comment by Kahonnohak** [comment]
@sirius651 

Thanks but I wanted to make the image fix on the left so the graphic image doesn’t bother too much with text space. So the right side would be trimmed as the width gets narrower.
 
 Other things
 -  Please add the link to the risk warning. `https://docs.algem.io/algem-protocol/risk-management`
 - Sorry but  please show the underbar as default (mobile users would not see it otherwise)

**Comment by sirius651** [comment]
> @sirius651
> 
> Thanks but I wanted to make the image fix on the left so the graphic image doesn’t bother too much with text space. So the right side would be trimmed as the width gets narrower. (please see this: https://algem-lending-dev.vercel.app/)
> 
> Other things
> 
> * Please add the link to the risk warning. `https://docs.algem.io/algem-protocol/risk-management`
> * Sorry but  please show the underbar as default (mobile users would not see it otherwise)

made a quick fix. regarding image, set position left from center. can you check again? @Kahonnohak

**Comment by Kahonnohak** [comment]
Thank you look great! I'll ask Shun to review.

**Comment by sirius651** [comment]
@niklabh could you help why rpc-tests(shiden) is not successful?

**Comment by niklabh** [comment]
fixed by rerun

**Comment by sirius651** [comment]
@Kahonnohak everything is on ready

**Review by Kahonnohak — APPROVED** [review]
Finally! LGTM!

**Pull request merged**